src/scheduler/drr.rs

```diff
@@ -103,7 +107,10 @@ impl<const HASH_PORTS: bool> Scheduler for Drr<HASH_PORTS> {
     }
 
     fn deq(&mut self) -> Result<Option<Pkt>, Report> {
-        let curr_tot_qsize: usize = self.curr_qsizes.iter().sum();
+        let mut curr_tot_qsize: usize = 0;
+        for i in 0..32 {
+            curr_tot_qsize += self.queues[i].len();
+        }
```

Comment:
Why is `self.curr_qsizes.iter().sum()` wrong?

Reply:
The idea of `curr_qsizes` is that here: https://github.com/akshayknarayan/hwfq/pull/4/files#diff-d2ab1fc19bb5551791b063c1264b142fb50264ed9ee5b8f383ebdd0aca2c1237R93 and here: https://github.com/akshayknarayan/hwfq/pull/4/files#diff-d2ab1fc19bb5551791b063c1264b142fb50264ed9ee5b8f383ebdd0aca2c1237R132, we keep a queue size counter.